docs/csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_4.cs

```diff
@@ -1,68 +1,57 @@
-class ArrayClass
+using System;
+
+class ArrayExample
 {
     static void PrintArray(string[] arr)
     {
         for (int i = 0; i < arr.Length; i++)
         {
-            System.Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
+             Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
         }
-        System.Console.WriteLine();
+        Console.WriteLine();
     }
 
     static void ChangeArray(string[] arr)
     {
-        // The following attempt to reverse the array does not persist when
-        // the method returns, because arr is a value parameter.
-        arr = (arr.Reverse()).ToArray();
-        // The following statement displays Sat as the first element in the array.
-        System.Console.WriteLine("arr[0] is {0} in ChangeArray.", arr[0]);
+        // // Reversing the array persists when the method returns.
```

Comment:
nit: too many `//`
Also consider rephrasing: *Reversing the array persists...* -> *The array stays reversed...*